cspell.json

```diff
@@ -69,6 +69,7 @@
     "healthcheck",
     "HSTS",
     "huskyrc",
+    "immer",
```

Comment:
Using immer for deep copying, as well as some test constructions. Used in `main-stbr`